src/supplemental/nanolib/parquet/parquet.cc

```diff
@@ -821,7 +821,6 @@ parquet_read(conf_parquet *conf, char *filename, vector<uint64_t> keys)
 			int64_t values_read = 0;
 			int64_t rows_read   = 0;
 			int16_t definition_level;
-			int16_t repetition_level;
```

Comment:
is it useful ? @xinyi-xs

Reply:
It may not be useful now, but it could come in handy in the future.